### PR TITLE
Fix list mapping collection #533

### PR DESCRIPTION
**Pull request by ozrendev**

### CODE REVIEW

**Comment by facundoolano** [comment]
Can you add a test to check for those collections? (so it fails next time this happens)

**Comment by ozrendev** [comment]
Hey, I added a check inside the function to throw an error if it returns empty. 

Is this what you mean or within the list of unit tests?

**Comment by facundoolano** [comment]
I mean in the unit tests

**Comment by ozrendev** [comment]
When I looked further, it seems that the mappings will change depending on if you make a request with or without a category. I've made a fix for this and added tests if you could review it and let me know if this is the correct way to do this.

File: lib/list.js [hunk 1]
```diff
@@ -82,17 +82,17 @@ const COLLECTIONS_MAPPING = ['ds:3', 0, 1];
...
-  GROSSING: 'top grossing apps',
-  TOP_FREE_GAMES: 'top free games',
-  TOP_PAID_GAMES: 'top paid games',
-  TOP_GROSSING_GAMES: 'top grossing games',
-  TRENDING: 'trending',
-  NEW_FREE: 'top new free android apps',
-  NEW_PAID: 'top new paid android apps',
-  NEW_FREE_GAMES: 'top new free games',
-  NEW_PAID_GAMES: 'top new paid games'
+  TOP_FREE: ['top free', 'top free apps'],
```
Comment (facundoolano) [inline lib/list.js:85]: can you explain this change better, from the perspective of the user of this library? why do we need TOP_FREE to include these two cases? shouldn't it also include 'top free games' for that matter?

**Comment by facundoolano** [comment]
Closing as other changes have been merged in this area.

**Pull request closed without merging**